de_web_plugin.cpp

```diff
@@ -3504,6 +3559,15 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
                             else if (modelId.isEmpty() && j->id() == 0x0005) // model id
                             {
                                 modelId = j->toString().trimmed();
+                                // replace ORVIBO model ID
```

Comment:
I'll merge this for 2.05.67 but suggest we refactor it later, so that optional translations or mappings of device model identifiers into readable text are kept in the device map.